db/schema.rb

```diff
@@ -137,6 +137,7 @@
     t.integer "working_days_submission_to_started"
     t.integer "working_days_since_started"
     t.boolean "induction_required"
+    t.text "recommendation_assessor_note"
```

Comment:
This doesn’t match the migration, it needs to be defined with the same `null` and `default` arguments.